src/Microsoft.SqlTools.ManagedBatchParser/Localization/sr.cs

```diff
@@ -0,0 +1,6663 @@
+        }
+
+        public static string SchemaHierarchy_ColumnSetLabelWithTypeAndKeyString
+        {
+            get
+            {
+                return Keys.GetString(Keys.SchemaHierarchy_ColumnSetLabelWithTypeAndKeyString);
+            }
+        }
+
+        public static string UniqueIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.UniqueIndex_LabelPart);
+            }
+        }
+
+        public static string NonUniqueIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.NonUniqueIndex_LabelPart);
+            }
+        }
+
+        public static string ClusteredIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.ClusteredIndex_LabelPart);
+            }
+        }
+
+        public static string NonClusteredIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.NonClusteredIndex_LabelPart);
+            }
+        }
+
+        public static string History_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.History_LabelPart);
+            }
+        }
+
+        public static string SystemVersioned_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.SystemVersioned_LabelPart);
+            }
+        }
+
+        public static string External_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.External_LabelPart);
+            }
+        }
+
+        public static string FileTable_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.FileTable_LabelPart);
+            }
+        }
+
+        public static string DatabaseNotAccessible
+        {
+            get
+            {
+                return Keys.GetString(Keys.DatabaseNotAccessible);
+            }
+        }
+
+        public static string ScriptingParams_ConnectionString_Property_Invalid
+        {
+            get
+            {
+                return Keys.GetString(Keys.ScriptingParams_ConnectionString_Property_Invalid);
+            }
+        }
+
+        public static string ScriptingParams_FilePath_Property_Invalid
+        {
+            get
+            {
+                return Keys.GetString(Keys.ScriptingParams_FilePath_Property_Invalid);
+            }
+        }
+
+        public static string ScriptingListObjectsCompleteParams_ConnectionString_Property_Invalid
+        {
+            get
+            {
+                return Keys.GetString(Keys.ScriptingListObjectsCompleteParams_ConnectionString_Property_Invalid);
+            }
+        }
+
+        public static string StoredProcedureScriptParameterComment
+        {
+            get
+            {
+                return Keys.GetString(Keys.StoredProcedureScriptParameterComment);
+            }
+        }
+
+        
```

Comment:
I have optimized the String resource files by removing unused keys in Localization\SR.Strings. Now the Sr.cs files contain only needed strings used by the project.

Reply:
@NiranjanVirtuosity I can't see where you're removing the batch parser string resources from the ServiceLayer project.  Should we do that as well if they are no longer needed there?

Reply:
@kburtram I have removed the unused strings from both ServiceLayer and ManagedBatchParser projects

Reply:
@NiranjanVirtuosity hmm, I still see `BatchParser_CommentNotTerminated` in the ServiceLayer project in your PR.  I'd expect this and other BatchParser string to be removed since they are now duplicated in the new project.  Do you know why these strings need to be duplicated in both projects?

Reply:
Yes I removed those in serviceLayer now.